README.md

```diff
@@ -1,239 +1,224 @@
 # Ecma TC39 JavaScript Decimal proposal
 
-The TC39 Decimal proposal is an investigation into adding a built-in data type in JavaScript to represent base-10 decimal numbers.
+The TC39 Decimal proposal is an investigation into adding a
+built-in data type in JavaScript to represent base-10
+decimal numbers.
 
-This whole proposal is basically a big open question, and we'd welcome your participation in discussing the design space in the issues linked above. **We are seeking input for your needs around JavaScript decimal in [this survey](https://forms.gle/A2YaTr3Tn1o3D7hdA).**
+This whole proposal is a big open question, and we’d welcome your participation in discussing the design space in the issues linked above. **We are seeking input for your needs around JavaScript decimal in [this survey](https://forms.gle/A2YaTr3Tn1o3D7hdA).**
 
-**Champions**: Philip Chimento (Igalia), Andrew Paprocki (Bloomberg), Jesse Alama (Igalia)
+**Champions**:
+
++ Philip Chimento (Igalia)
++ Andrew Paprocki (Bloomberg)
++ Jesse Alama (Igalia)
```

Comment:
Go ahead and move yourself to the top of the list, so that it's clear who to contact first :smile:

Reply:
Happy to take the heat -- thanks!